rpcs3/util/console.cpp

```diff
@@ -0,0 +1,41 @@
+#ifdef _WIN32
+#include "console.h"
+#include "Windows.h"
+#include <stdio.h>
+
+namespace utils
+{
+	void attach_console(int stream, bool open_console)
+	{
+		if (!stream)
+		{
+			return;
+		}
+
+		if (!AttachConsole(ATTACH_PARENT_PROCESS))
```

Comment:
The original starement was OR because there might nly be a parent process console, in which case AllocConsole is called. This is AND-like implementation which is wrong.